index.html

```diff
@@ -5583,8 +5583,8 @@ <h2>Element Interaction</h2>
  set the <a>checkedness</a> of the element to true
  if the element has a <a><code>checked</code></a> content attribute and false if it does not,
  empty the list of <a>selected files</a>,
- and then invoke the <a>value sanitization algorithm</a>,
- if the <a><code>type</code></a> attribute’s current state defines one.
+ and then invoke the <a>value sanitization algorithm</a>
+ iff the <a><code>type</code></a> attribute’s current state defines one.
```

Comment:
Please note the double `ff` here for if.

Reply:
I mean “iff”: https://en.wikipedia.org/wiki/If_and_only_if

Reply:
Never heard of that. Thanks!